pkg/graph/schema/types/model_to_operations_common_milestone.graphql

```diff
@@ -0,0 +1,16 @@
+type ModelToOperationsCommonMilestone {
+  id: UUID!
+  categoryName: String!
+
+  createdBy: UUID!
+  createdByUserAccount: UserAccount!
+  createdDts: Time!
+  modifiedBy: UUID
+  modifiedByUserAccount: UserAccount
+  modifiedDts: Time
+}
+
+extend type Query {

```

Comment:
I don't think this needs to be exposed at the query level, I think we have it on the models to operation matrix, so we can get the model plan id from there

Reply:
Correct -- we actually (currently) would want to avoid exposing this at anything other than at the level of a specific Model Plan, because properties like `isSuggested` / `isAdded` won't be resolvable